src/test/e2e/services/user-service.e2e.test.ts

```diff
@@ -95,8 +115,9 @@ afterAll(async () => {
     await db.destroy();
 });
 
-afterEach(async () => {
+beforeEach(async () => {
```

Comment:
it's a better practice to clean data before not after since it allows debugging failed tests